README.md

```diff
@@ -1,18 +1,40 @@
 
-# Common Hosted Form Service [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+# Common Hosted Form Service [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE) [![Quality Gate](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/api/badges/gate?key=common-hosted-form-service-master)](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-master)
+
+[![Bugs](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/api/badges/measure?key=common-hosted-form-service-master&metric=bugs)](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-master)
+[![Vulnerabilities](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/api/badges/measure?key=common-hosted-form-service-master&metric=vulnerabilities)](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-master)
+[![Code Smells](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/api/badges/measure?key=common-hosted-form-service-master&metric=code_smells)](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-master)
+[![Coverage](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/api/badges/measure?key=common-hosted-form-service-master&metric=coverage)](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-master)
+[![Lines](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/api/badges/measure?key=common-hosted-form-service-master&metric=lines)](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-master)
+[![Duplication](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/api/badges/measure?key=common-hosted-form-service-master&metric=duplicated_lines_density)](https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-master)
 
 Create, edit and publish forms.
 
 ## Directory Structure
 
     .github/                   - PR and Issue templates
+    app/                       - Application Root
+    ├── frontend/              - Frontend Root
+    │   ├── src/               - Vue.js frontend web application
+    │   └── tests/             - Vue.js frontend web application tests
+    ├── src/                   - Node.js backend web application
+    │   ├── db/migrations      - data migration scripts
+    │   ├── docs/              - OpenAPI 3.0 Specification
+    │   └── forms/             - Models, Controllers, Routes for the forms
```

Comment:
THere's a routes folder here too but not sure if this is just a quick overview that doesn't need all folders

Reply:
I didn't want to add too many folders just yet in case things were still in flux. This is just meant to be a quick overview for folders that may not be as obvious for what they contain and do.